libuuu/fastboot.cpp

```diff
@@ -310,11 +314,13 @@ int FBCopy::run(CmdCtx *ctx)
 		nt.total = buff->size();
 		call_notify(nt);
 
-		for (i = 0; i < buff->size(); i += this->m_Maxsize_pre_cmd)
+		for (i = 0;
```

Comment:
m_Maxsize_pre_cmd need be update at beginning, such as construct function or at parser function.  so needn't change below code.

Reply:
the code is needed here because we have to know about the packet size to split the content, using construct fonction or at parser level will imply lot of modifications since there is no way to go from parser to low level constructor, doing so will require modify lot of API... I didn't wanted to go that way because it overcomplicate the process, while a configuration class singleton store the configuration in a dedicated place and is easely integrated